src/app/components/MergeRow.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { c } from 'ttag';
-import { Icon } from 'react-components';
+import { useModals, Icon } from 'react-components';
+
+import MergeModal from './merge/MergeModal';
+
+const MergeRow = ({ style, mergeableContacts, userKeysList }) => {
+    const { createModal } = useModals();
 
-const MergeRow = ({ style }) => {
     const handleClick = () => {
-        // TODO start merge process
+        createModal(<MergeModal contacts={mergeableContacts} userKeysList={userKeysList} />);
```

Comment:
`onMerge` not defined?

Reply:
No, the merge logic requires the user to select which contacts will be marked for merging and deletion, which will only happen inside the modal (that has two steps)

Reply:
I misunderstood your point probably. There's an `onMerge` now